CompetitionTemplate/src/competition/subsystems/drive/commands/DriveToDistanceCommand.java

```diff
@@ -0,0 +1,70 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.drive.PoseSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class DriveToDistanceCommand extends BaseCommand {
```

Comment:
Worth seeing if we can use the one from the low bar feature branch instead (and maybe spruce it up)